taxi/models.py

```diff
@@ -15,7 +18,13 @@ def __str__(self):
 
 
 class Driver(AbstractUser):
-    license_number = models.CharField(max_length=255, unique=True)
+    license_number = models.CharField(max_length=8, unique=True)
+
+    def clean_number(self):
+        if not re.match(r"^[A-Z]{3}\d{5}$", self.license_number):
+            raise ValidationError(
+                "License number must consist of 3 "
+                "uppercase letters followed by 5 digits.")
```

Comment:
The method `clean_number` is intended to validate the `license_number` field. However, it should be named `clean_license_number` to be automatically called during the model's full_clean process. Consider renaming this method to ensure proper validation integration.